Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -2,87 +2,104 @@ import Foundation
 import CoreLocation
 import MapboxDirections
 
-// MARK: - RouteController
+// MARK: Parameterizing RouteController
+
+// MARK: Rerouting logic
```

Comment:
Marks don’t nest, but we can do something combined like “RouteController Rerouting”, then put a `MARK: -` where non-RouteController stuff begins.

Reply:
Tried to rework those sections a bit